Web Forum/Areas/Identity/Pages/TopicPage/TopicList.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@page "/TopicList"
+@model Web_Forum.Views.Home.TopicPage.TopicList
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>Topics</h1>
+
+<div>
+    @if (@Model.LoggedUser() != null)
+    {
+        <button class="nav-link text-dark" onclick="location.href='./AddTopic'">Add topic</button>
+    }
+</div>
+
+<div class="py-5 bg-light">
+    <div class="container">
+        <div class="row">
+            @if (Model.Topics != null)
+            {
+                foreach (var topic in Model.Topics)
+                {
+                //version with coments
```

Comment:
Do you need this commented block?

Reply:
Well, if we don't include topics commentary it isn't needed anymore.